### ISSUE TITLE
`ArgumentException` on app launch

### ISSUE THREAD

**Issue by Lamparter** [josephbeattie/Fluent-Media-Player-Dev#262]
> Hi, @Lamparter , I forgot to say that I was able to complete the initial setup. After that, this is what I keep getting:

> https://github.com/user-attachments/assets/cb9ad5ac-8e8e-4864-9c88-22c8d7f41e49

_Originally posted by @vivadavid in https://github.com/Rise-Software/Rise-Media-Player/issues/256#issuecomment-2427822657_

**Comment by ghost** [comment]
yeah, the same is happening on my windows 11 notebook version 24H2 (samsung book 2 Pro (2022)), the application seems to have been abandoned for a long time...
